### PR TITLE
feat: add details about swap to success page

### LINKED ISSUE #106: "Voila" (Swap success page) should show details
### You sent `X` `coin_a` and received `Y` `coin_b`.
You paid `Y` `coin_b` in fees ( `Z` %) for this.

That would be great I think.

### PR DESCRIPTION
**Pull request by ImmanuelSegol**
closes #106

### CODE REVIEW

File: src/views/reverse/reverse.js [hunk 1]
```diff
@@ -90,7 +90,19 @@ class ReverseSwap extends React.Component {
...
               />
-              <StepsWizard.Step num={4} render={() => <Confetti />} />
+              <StepsWizard.Step
+                num={4}
+                render={() => (
+                  <Confetti
+                    notifie={style => (
+                      <span className={style}>
+                        You sent {swapInfo.baseAmount} {swapInfo.base} and
+                        received {swapInfo.quote} {swapInfo.quoteAmount}
```
Comment (michael1011) [inline src/views/reverse/reverse.js]: It should be `${amount} ${asset}` all time

File: src/components/confetti/index.js [hunk 2]
```diff
@@ -12,11 +12,15 @@ const confettiStyles = () => ({
     justifyContent: 'center',
   },
   text: {
-    fontSize: '30px',
+    fontSize: '40px',
+    fontWeight: 300,
+  },
+  notifie: {
+    fontSize: '25px',
```
Comment (michael1011) [inline src/components/confetti/index.js]: Could you make the font of this class a little more lighter to differentiate if from the "Swap successful" text?

File: src/components/confetti/index.js [hunk 3]
```diff
@@ -12,11 +12,15 @@ const confettiStyles = () => ({
...
   },
 });
 
-const StyledConfetti = ({ classes }) => (
+const Confetti = ({ classes, notifie }) => (
```
Comment (michael1011) [inline src/components/confetti/index.js]: Why name the variable `notifie`?

Comment (ImmanuelSegol) [reply src/components/confetti/index.js]: I didn't think of something better, any text user the success message should give the user information relevant to them at the end of the process.

File: src/views/swap/swap.js [hunk 4]
```diff
@@ -76,7 +76,19 @@ const Swap = ({
...
             />
-            <StepsWizard.Step num={4} render={() => <Confetti />} />
+            <StepsWizard.Step
+              num={4}
+              render={() => (
+                <Confetti
+                  notifie={style => (
+                    <span className={style}>
+                      You sent {swapInfo.baseAmount} {swapInfo.base} and
+                      received {swapInfo.quote} {swapInfo.quoteAmount}
```
Comment (michael1011) [inline src/views/swap/swap.js]: Same here: should be `${amount} ${asset}`

**Pull request merged**